docs/testcases/UI/Billing/762/762_7.md

```diff
@@ -0,0 +1,47 @@
+# Check Storages quota for Billing Center
+
+Test verifies that
+- billing quota with action *Read-only mode* for Storages specified for ***Billing Center*** is applied only to non-admin user
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+- *Billing center* (`<Billing_center>`) includes non-admin user and admin user from the Prerequisites
+- Billing values:
+    - *Storages* billing for the `<Billing_center>` from the Prerequisites for the current quarter is `<billing1>`
+- Perform [762_1](762_1.md) case
+
+**Preparations**:
+1. Login as admin user from the prerequisites
+2. Create 2 *object storages* (`storage1`, `storage2`) and *fs storage* (`fs_storage`). Give all permissions to these storages to non-admin user from the Prerequisites
+3. Logout. Login as non-admin user from the prerequisites
+4. Open the **Tools** page
+5. Select test tool
+6. Launch a selected tool with default settings
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as admin user from the prerequisites | |
+| 2 | Open the **Billing** page | |
+| 3 | Click **Storages** tab in the **Quotas** section |  |
+| 4 | Click **Add quota** button for ***Billing centers*** section | ***Create storages quota*** modal opens that contains: <li> ***Billing center*** field <li> ***Quota*** field, <li> ***Quota period*** dropdown list with values `per month`, `per quarter`, `per year` (default value is `per month`) <li> ***Actions*** section that contains ***Threshold*** and ***Action*** fields and **Add Action** button <li> **Cancel** and **Save** buttons. |
+| 5 | Click into the ***Billing center*** field and select `<Billing_center>` from the prerequisites | | 
+| 6 | Specify follow values for quota <li> *Quota* - `<billing1>` <li> quota period -`per quarter` <li> *Threshold* - `70` <li> *Action* - `Read-only mode` | |
+| 7 | Click **Save** button | The following info appears in the ***Overall*** section: <li> `<Billing_center>: <quota>$ per quarter`, where `<quota>` is value specified at step 5 <li> `70%: read-only mode` - pair of threshold-action configured at step 7-9 |
+| 8 | Wait time equals to `<billing_quotas_period>` from the [762_1](762_1.md) case Preparations and refresh page | Yellow circle is shown near the quota created at step 7|
+| 9 | Open the **Library** page | |
+| 10 | Open the `storage1` created at step 2 of Preparations | Storage page opens and is available for editing |
+| 11 | Create `file1.txt` | | 
+| 12 | Open the `fs_storage` created at step 2 of Preparations | Storage page opens and is available for editing |
+| 13 | Logout | |
+| 14 | Login as non-admin user from the prerequisites | |
+| 15 | Repeat steps 10-11, 13 | For both cases Storage page opens and doesn't contain <li> **Create** and **Upload** buttons for whole table <li> **Edit** and **Remove** buttons for each row |
```

Comment:
Fix `steps 10-11, 13` to `steps 9-10, 12`

Reply:
Done